core/backend/src/test/integration/Agent.test.ts

```diff
@@ -23,8 +23,8 @@ describe("Agent iModel Download (#integration)", () => {
     // IModelTestUtils.setupDebugLogLevels();
 
     const agentConfiguration: AgentAuthorizationClientConfiguration = {
-      clientId: Config.App.getString("imjs_agent_test_client_id"),
-      clientSecret: Config.App.getString("imjs_agent_test_client_secret"),
+      clientId: process.env.imjs_agent_test_client_id ?? "",

```

Comment:
Same goes for a handful of these, mind throwing when they're not set.

Reply:
For sure. Similar question as above, if you can think of a more elegant solution than this that is still descriptive of which item is undefined, let me know.
```
if (process.env.imjs_oidc_ulas_test_client_id === undefined)
  throw new Error("Could not find imjs_oidc_ulas_test_client_id");
if (process.env.imjs_oidc_ulas_test_redirect_uri === undefined)
  throw new Error("Could not find imjs_oidc_ulas_test_redirect_uri");
if (process.env.imjs_oidc_ulas_test_scopes === undefined)
  throw new Error("Could not find imjs_oidc_ulas_test_scopes");
```
^This is for a different file but same premise.